Project-2/README.md

```diff
@@ -12,20 +12,45 @@ Tip #4: Work together as a team.
 1. Let V be a vocabulary of ALCI consisting of a role name "P". Interpret part_of as "x is a part of y". Using this role name, define the following formulas in this language:
 ```
   (a)  PP that says that x is a proper part of y
+  PP ≡ P ⊓ ¬P¯
+  
   (b)  iPP that says that y is a proper part of x
+  iPP ≡ PP¯ =>
```

Comment:
Which is your answer?

Reply:
I believe the answer is: iPP ≡ (P ⊓ ¬P¯)¯
From the google doc: "In mereology, x is a proper part of y when x is a part of y and y is not a part of x (they are not the same entity)."

The statement above (should) translate accordingly, such that iPP that says that y is a proper part of x if x is a part of y (P) and (⊓) y is not a part of x (the inverse of P is negated).  (Is this a decent translation?)